### PR TITLE
Feature/subscribe-form

### PR DESCRIPTION
**Pull request by jacksmolny**

### CODE REVIEW

**Review by lazio — CHANGES_REQUESTED** [review]
https://app.clickup.com/t/7am0dd

point 2 from the task is not done

File: src/scenes/BlogPage/components/Previews/components/Subscribe/Subscribe.module.scss [hunk 1]
```diff
@@ -57,13 +57,17 @@ $layoutSwitchResolution: 1100px;
...
-  @include hl-padding(bottom, 60px);
-  @include hl-padding(top, 50px);
+  @include hl-padding(bottom, 45px);
+  @include hl-padding(top, 45px);
   @include hl-padding(right, 100px);
   @include hl-padding(left, 100px);
   display: flex;
   justify-content: space-between;
   background: linear-gradient(88.91deg, #3719ca 0.16%, #8161fe 99.8%);
+  & > form {
```
Comment (lazio) [inline src/scenes/BlogPage/components/Previews/components/Subscribe/Subscribe.module.scss:67]: can we use only class selectors? 
that way we're always sure we do not override styles for other elements

File: src/scenes/BlogPage/components/Previews/components/Subscribe/Subscribe.module.scss [hunk 2]
```diff
@@ -89,69 +93,133 @@ $layoutSwitchResolution: 1100px;
...
 
 .button {
   border: none;
   background: none;
   padding: 5px 0 0;
   cursor: pointer;
+  font-size: 0;
+  line-height: 0;
+
+  & > svg {
```
Comment (lazio) [inline src/scenes/BlogPage/components/Previews/components/Subscribe/Subscribe.module.scss:189]: svg is the only place where it's ok to use tag selectors

File: src/scenes/BlogPage/components/Previews/components/Subscribe/Subscribe.module.scss (hunk 1 above)
Comment (jacksmolny) [reply src/scenes/BlogPage/components/Previews/components/Subscribe/Subscribe.module.scss]: fixed

File: src/scenes/BlogPage/components/Previews/components/Subscribe/Subscribe.module.scss (hunk 2 above)
Comment (jacksmolny) [reply src/scenes/BlogPage/components/Previews/components/Subscribe/Subscribe.module.scss]: fixed

**Review by lazio — APPROVED** [review]

**Pull request merged**